### PR TITLE
fill set_stream funciton 

### PR DESCRIPTION
**Pull request by tingxingdong**
-  add stream to every kernel launching
-  passed all the tests on rocm 1.7

### CODE REVIEW

File: library/src/include/transform.h [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
 struct rocfft_execution_info_t
 {
-	void *workBuffer;
-	size_t workBufferSize;
-
-	rocfft_execution_info_t() : workBuffer(nullptr), workBufferSize(0)
-	{}
+    void *workBuffer;
+    size_t workBufferSize;
+    hipStream_t rocfft_stream = 0;//by default it is stream 0
```
Comment (bragadeesh) [inline library/src/include/transform.h]: default must be initialized in the constructor initializer list

Comment (bragadeesh) [reply library/src/include/transform.h]: hmm, i see c++ 11 allows this, never mind

**Comment by tingxingdong** [comment]
I see the cross sign" The commit cannot be build". It is the jenkins machine down.

File: library/src/include/transform.h (hunk 1 above)
Comment (tingxingdong) [reply library/src/include/transform.h]: Yes, the rocBLAS also use this feature. 
https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/library/src/include/handle.h#L31

**Comment by bragadeesh** [comment]
@tingxingdong do you have some test code or example to show this works?

**Comment by tingxingdong** [comment]
You mean,  set the stream? I think we can make one from the existing sample.

**Pull request merged**